mappings/net/minecraft/client/util/math/Matrix3f.mapping

```diff
@@ -8,10 +12,10 @@ CLASS net/minecraft/class_4581 net/minecraft/util/math/Matrix3f
 		ARG 1 matrix
 	METHOD <init> (Lnet/minecraft/class_4581;Z)V
 		ARG 1 matrix
-		ARG 2 loopCopy_BrokenDoesNothing
+		ARG 2 transposeVarArg
```

Comment:
`VarArg`?

Reply:
This is for transpose usage, but this arg is not properly passed to the constructor it delegates to (true instead of this param is used, but luckily this param is always called with true so no error emerges)

Reply:
I understand, but how does `VarArg` mean that the parameter is not being passed? `To me `VarArg` would mean it's a var arg (`boolean... transpose`).

> no error emerges

Someone creating a `new Matrix3f(..., false)` would be surprised when they find out that their matrix is being transposed even though they passed false to the constructor. I think the name should make it clear that it's broken, so that someone doesn't accidentally use it. Something like `IgnoredAlwaysTrue` would work.